tweaks/_main_code.scss

```diff
@@ -56,7 +56,7 @@
 
     div[class*="guildsWrapper-"] {
         width: 100%;
-        height: 75px;
+        height: 100px;
```

Comment:
Height should only change from 75px to 100px if `$server-list-above-enable-scrollbar` is set.

Reply:
Also, a 25px increase is a *lot*. Without needing a scrollbar myself, it looks very jarring. Does the scrollbar really have to be so thick? An approach similar to the channel scrollbar is likely better (way smaller, only show when hovering, have mousewheel do the work).